src/Stores/SubjectsPromises/selectors.js

```diff
@@ -0,0 +1,10 @@
+/* @flow */
+import type { State } from '../../types';
+
+export const subjectsPromisesSelector = (state: State) => {
```

Comment:
Would it make sense to write this a bit more functional?

```
export const subjectsPromisesSelector = (state: State) => (
state.subjectsPromises.subjectsPromises ? state.subjectsPromises.subjectsPromises : null
)
```

Same with the other selector here

Reply:
Sure.

I was anticipating having to derive some field properties here.

But you are correct. Lets write code for the now instead of _possible future requirementz_